scripts/zones/Bastok_Markets_[S]/IDs.lua

```diff
@@ -24,6 +24,7 @@ zones[xi.zone.BASTOK_MARKETS_S] =
         HUNT_RECORDED                 = 7745,  -- You record your hunt.
         OBTAIN_SCYLDS                 = 7747,  -- You obtain <number> [scyld/scylds]! Current balance: <number> [scyld/scylds].
         HUNT_CANCELED                 = 7751,  -- Hunt canceled.
+        CAMPAIGN_RESULTS_TALLIED      = 11763, -- Campaign results tallied.
```

Comment:
Can you align this with the other messages, otherwise this will get changed possibly for no reason when we run our monthly update

Reply:
Sorry, the intention was to make it align for the monthly update. What is wrong with it?
```xml
<thing type="DialogTableEntry">
  <field name="index">11763</field>
  <field name="text">Campaign results tallied.≺Prompt≻</field>
</thing>
```

Reply:
7751, 11763, 10844 is not ordered, our tooling will fix it and provide a false positive that there's been shifts

Reply:
Oops, my bad.